### PR TITLE
feat: parse CLI arguments from config file

### LINKED ISSUE #193: Config file support

### PR DESCRIPTION
**Pull request by bstrie**
Closes #193 

Signed-off-by: bstrie <865233+bstrie@users.noreply.github.com>

### CODE REVIEW

**Review by rvolosatovs — APPROVED** [review]
It's a shame that there's no functionality in `clap` built-in for doing this.
Minor clippy lint, which we can fix later. LGTM, thanks!

File: src/main.rs [hunk 1]
```diff
@@ -1,10 +1,12 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: AGPL-3.0-only
 
-use std::fs::File;
+use std::error::Error;
```
Comment (rvolosatovs) [inline src/main.rs:4]: If you're fixing the clippy lint, then also please import this via the full path (`std::error::Error`) at call site, which is the convention used currently throughout the code base to disambiguate (e.g. `anyhow::Error`)

Comment (bstrie) [reply src/main.rs:4]: Sure, I can do that quickly.

**Review by rvolosatovs — APPROVED** [review]
LGTM, thanks!

**Pull request merged**